packages/components/src/SpectrumUtils.test.ts

```diff
@@ -0,0 +1,22 @@
+import { defaultTheme } from '@adobe/react-spectrum';
+import { themeDHDefault } from './SpectrumUtils';
+
+describe('themeDHDefault', () => {
+  it('should merge Spectrum default with DH custom styles', () => {
+    const { global, light, dark, medium, large } = defaultTheme;
+
+    expect(themeDHDefault).toEqual({
+      global,
+      light: {
+        ...light,
+        'dh-spectrum-theme--light': 'mock.light',
```

Comment:
You should be able to remove the mock and just expect this to equal `dh-spectrum-theme--light`. The css identity-obj-proxy just creates an identity object. I.e. returns the requested key as the value for anything you try to pull from the object. Might be `--dh-spectrum-theme--light`, but it shouldn't really need the mock

Reply:
The issue is that the implementation doesn't actually request any specific keys. It use a spread operator which seems to yield an empty object from the proxy. The test will pass, but it won't actually prove the dh props got added.